deploy/firesim

```diff
@@ -119,6 +119,9 @@ def buildafi(globalbuildconf):
 
     auto_create_bucket(globalbuildconf.s3_bucketname)
 
+    #check to see email notifications can be subscribed
+    get_snsname_arn()
```

Comment:
This is a great improvement but we could still have the case where the user has `sns:CreateTopic` but they are missing the `sns:Publish` permission and thus still can't send the message.  It's different than the failure case @rahulb10 and I hit but it we should put try/catch blocks around the other AWS API calls

Reply:
My concern here is addressed by the changes in b25cd16